pkg/controller/pod_volume_backup_controller.go

```diff
@@ -120,22 +139,29 @@ func (c *podVolumeBackupController) processQueueItem(key string) error {
 		return nil
 	}
 
-	// only process items for this node
-	if req.Spec.Node != c.nodeName {
-		return nil
-	}
-
 	// Don't mutate the shared cache
 	reqCopy := req.DeepCopy()
 	return c.processBackupFunc(reqCopy)
 }
 
-func (c *podVolumeBackupController) processBackup(req *arkv1api.PodVolumeBackup) error {
-	log := c.logger.WithFields(logrus.Fields{
+func loggerForPodVolumeBackup(baseLogger logrus.FieldLogger, req *arkv1api.PodVolumeBackup) logrus.FieldLogger {
+	log := baseLogger.WithFields(logrus.Fields{
 		"namespace": req.Namespace,
 		"name":      req.Name,
 	})
 
+	if len(req.OwnerReferences) == 1 {
+		log = log.WithField("arkBackup", req.OwnerReferences[0].Name)
```

Comment:
We've used `backup` as the key with `namespace/name` as the value before - let's be consistent?